docs/functions.rst

```diff
@@ -1573,6 +1573,43 @@ Example:
           name: "test-snapshot-content-content-dfc8fa67-8b11-4fdf-bf94-928589c2eed8"
 
 
+DeleteDataUsingKopiaServer
+--------------------------
+
+This function deletes the snapshot data backed up by the ``BackupDataUsingKopiaServer``
+function. It creates a new Pod that runs ``delete snapshot`` job.
+
+.. note::
+   The ``image`` argument requires the use of ``ghcr.io/kanisterio/kanister-tools``
+   image since it includes the required tools to delete snapshot in
+   the object store.
```

Comment:
```suggestion
   The ``image`` argument requires the use of ``ghcr.io/kanisterio/kanister-tools``
   image since it includes the required tools to delete snapshot from
   the object store.
```